Description: Support specifying origlibdirs for dependencies searching
 In order to improve the limited searching for dependencies in config_unix.py
 This adds support to specify which directories should be searched
 under localbase in the environment variable ORIGLIBDIRS.
 .
 The debian/rules will then ensure that the correct directories are
 searched by doing something like:
   ORIGLIBDIRS=/lib:/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH)
 .
Bug-Ubuntu: http://bugs.launchpad.net/bugs/755980
Author: Scott Moser <smoser@ubuntu.com>
Last-Update: 2011-06-20
--- a/config_unix.py
+++ b/config_unix.py
@@ -22,6 +22,9 @@
     origlibdirs = os.environ['ORIGLIBDIRS'].split(":")
 
 
+if 'ORIGLIBDIRS' in os.environ and os.environ['ORIGLIBDIRS'] != "":
+    origlibdirs = os.environ['ORIGLIBDIRS'].split(":")
+
 def confirm(message):
     "ask a yes/no question, return result"
     reply = raw_input('\n' + message + ' [Y/n]:')
